src/test/java/delete_task_use_case/DeleteTaskInteractorTest.java

```diff
@@ -0,0 +1,56 @@
+package delete_task_use_case;
+
+import data_access.DataAccessDay;
+import data_access.DataAccessEvent;
+import data_access.DataAccessTask;
+
+import java.util.HashMap;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class DeleteTaskInteractorTest {
+    private final static DeleteTaskInputData inputData1 = new DeleteTaskInputData(
+            5, "Task");
+    private final static DeleteTaskInputData inputData2 = new DeleteTaskInputData(
+            0,"Task2");
+    private final static DeleteTaskDataAccess dataAccess = new DeleteTaskDataAccess();
+    private final static DeleteTaskDsGateway dsGateway = dataAccess;
+    private final static DeleteTaskPresenter outputBoundary = new DeleteTaskPresenter();
+
+    @Test
+    void testDeleteTaskSuccess() {
+        DataAccessTask task = new DataAccessTask("Task");
+        HashMap<String, DataAccessTask> tasks = new HashMap<>();
+        HashMap<String, DataAccessEvent> events = new HashMap<>();
+        tasks.put("Task", task);
+        DataAccessDay day = new DataAccessDay(tasks, events);
+        dataAccess.getDays().set(5, day);
+
+        DeleteTaskInputBoundary inputBoundary = new DeleteTaskInteractor(outputBoundary, dsGateway);
+        DeleteTaskOutputData outputData = inputBoundary.deleteTask(inputData1);
+
+        assertEquals(5, outputData.getDayIndex());
+        assertEquals("Task", outputData.getTaskTitle());
+        assertTrue(day.getTasks().isEmpty());
```

Comment:
Again, you should check if the Day object at index 5 still has a Task object with the name "Task".

Reply:
Added the change suggested.